_flake8_tergeo/checks/ast_raise.py

```diff
@@ -50,3 +51,46 @@ def _check_raise_from_itself(node: ast.Raise) -> IssueGenerator:
             issue_number="044",
             message="Found exception raised from itself.",
         )
+
+
+def _get_except(node: ast.AST) -> ast.ExceptHandler | None:
+    current: ast.AST | None = node
+    while current:
+        parent = get_parent(current)
+        if isinstance(parent, ast.ExceptHandler):
+            return parent
+        current = parent
+    return None
+
+
+def _get_129_issue(node: ast.Raise) -> Issue:
+    return Issue(
+        line=node.lineno,
+        column=node.col_offset,
+        issue_number="129",
+        message="The cause of the raised error is the same as the caught exception.",
+    )
+
+
+def _check_raise_caught_class(node: ast.Raise) -> IssueGenerator:
+    if not node.cause or not isinstance(node.cause, (ast.Name, ast.Attribute)):
+        return
+
+    except_node = _get_except(node)
+    if not except_node:
```

Comment:
We want to check if except_node is None here, not for its truthiness in general, so I suggest wemake it explicit:
```suggestion
if except_node is None:
```

Reply:
In the whole code base, we never did this as ast nodes never overwrite \_\_bool\_\_